final/Containerfile

```diff
@@ -160,8 +160,6 @@ RUN set -ex \
 # Make a workdir usable by either the podman or root user
 RUN mkdir -p /workdir && chown podman:podman /workdir
 
-# Set the default user to podman, this will be overridden in the podman dev container
-USER podman
```

Comment:
Can we also remove the configuration of the podman user from the containerfile since we're not going to use it any more?

Reply:
Definitely :+1:

Reply:
@cidrblock Can you re-review this portion?

Reply:
Looks good